src/app/layout/footer/footer.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component } from '@angular/core';
+import { faFacebookF, faInstagram, faTwitter } from '@fortawesome/free-brands-svg-icons';
+
+@Component({
+  selector: 'app-footer',
+  templateUrl: './footer.component.html',
+  styleUrls: ['./footer.component.less']
```

Comment:
I know you are not using styles, but, for consistency, the extension of this file should be "scss", please change it. I know that the CLI generated the file like this, but this was due to a previous configuration, now on all style files will have the scss extension.

Reply:
Updated